spec/jekyll_seo_tag_spec.rb

```diff
@@ -247,6 +247,16 @@
           expect(output).to match(expected)
         end
       end
+
+      context 'with page.image' do
+        let(:site) { make_site('twitter' => site_twitter, 'url' => 'http://example.invalid') }
+        let(:page) { make_page('image' => 'foo.png') }
```

Comment:
If root, then you don't need `prepend: '/'` above. I'd have the test match your intended use, as right now, if I put `/img/foo.png`, it'd output as `http://exmaple.invalid//img/foo.png`, I believe.